apps/condo/domains/common/utils/GraphQLLoggerApp.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const gql = require('graphql-tag')
+const { get, set } = require('lodash')
+
+function normalizeQuery (string) {
+    if (!string) return ''
+    return string.replace(/[\s,]+/g, ' ').trim()
```

Comment:
it's better for performance to move static regexps to constants and reuse.

Reply:
you don't need to reuse it. it's better to store a full function context inside if it's possible or near the function it's increase the readability